orchagent/routeorch.cpp

```diff
@@ -371,6 +374,7 @@ bool RouteOrch::invalidnexthopinNextHopGroup(const NextHopKey &nexthop)
             return false;
```

Comment:
Is there a case where some were successful but then encountered an error?  In that case what should we do with 
gCrmOrch->decCrmResUsedCounter(CrmResourceType::CRM_NEXTHOP_GROUP_MEMBER); for the previous successful case? Should we restore it?

Reply:
This is returning false, so expectation is it will retry and gets removed in next cycle. Moreover this is the original code and prefer to leave it as it is.